### PR TITLE
chore(weave): add crown icon

### PR DESCRIPTION
**Pull request by mbarrramsey**
Screen Shot 2024-01-03 at 9 35 48 AM

Add crown-pro icon for account selector project (see [design](https://www.figma.com/file/m9twb1HoHsijlTlqVzRReW/Account-selector?type=design&node-id=4014-68961&mode=design&t=ZNF424zMDjGs0IH4-0)).

### CODE REVIEW

**Comment by mbarrramsey** [comment]
I have read the CLA Document and I hereby sign the CLA

File: weave-js/src/assets/icons/icon-crown-pro.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14" fill="none">
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M7.67082 2.5884C7.36331 2.0887 6.63697 2.0887 6.32946 2.5884L4.50416 5.55451L2.39956 4.50221C1.88694 4.2459 1.30974 4.7089 1.44875 5.26492L2.49237 9.43942C2.54106 9.63418 2.71605 9.77081 2.91681 9.77081H11.0835C11.2842 9.77081 11.4592 9.63418 11.5079 9.43942L12.5515 5.26492C12.6905 4.7089 12.1133 4.2459 11.6007 4.50221L9.49612 5.55451L7.67082 2.5884ZM5.20737 6.08133L7.00014 3.16808L8.79291 6.08133C9.0068 6.42889 9.45076 6.55547 9.81577 6.37296L11.595 5.48335L10.7419 8.89581H3.2584L2.40528 5.48335L4.18451 6.37296C4.54952 6.55547 4.99349 6.42889 5.20737 6.08133ZM3.20848 10.3542C2.96685 10.3542 2.77098 10.55 2.77098 10.7917C2.77098 11.0333 2.96685 11.2292 3.20848 11.2292H10.7918C11.0334 11.2292 11.2293 11.0333 11.2293 10.7917C11.2293 10.55 11.0334 10.3542 10.7918 10.3542H3.20848Z" fill="#6645D1"/>
```
Comment (jwlee64) [inline weave-js/src/assets/icons/icon-crown-pro.svg:2]: this needs to have fill="currentColor"

File: weave-js/src/assets/icons/icon-crown-pro.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14" fill="none">
```
Comment (jwlee64) [inline weave-js/src/assets/icons/icon-crown-pro.svg:1]: i also think that we normally want the svgs to be width 24 height 24 and viewbox 0 0 24 24, but maybe thats just not available

Comment (jwlee64) [reply weave-js/src/assets/icons/icon-crown-pro.svg:1]: theres a 24x24 one here

https://www.figma.com/file/01KWBdMZg5QM9SRS1pQq0z/Design-System----Robot-Styles?type=design&node-id=8245-7172&mode=design&t=KuG9RevV3OUG9Oon-4

Comment (jamie-rasmussen) [reply weave-js/src/assets/icons/icon-crown-pro.svg:1]: For consistency we usually ask the design team to export at 24x24 and share via their Google Drive folder. The addicon script in the devtools repo can then take care of things like updating the fill value for you.

**Review by jamie-rasmussen — APPROVED** [review]
Something's unhappy in CI, but doesn't seem related to this PR.

**Pull request merged**